docs/imviz/plugins.rst

```diff
@@ -135,6 +135,10 @@ an interactively selected region. A typical workflow is as follows:
 2. Draw a region over the object of interest (see :ref:`imviz_defining_spatial_regions`).
 3. Select the desired image using the :guilabel:`Data` dropdown menu.
 4. Select the desired region using the :guilabel:`Subset` dropdown menu.
+   You can use the :ref:`imviz-subset-plugin` plugin to center it first on the
+   object of interest using its center of mass, if you wish.
+   Depending on the object, that may take several interations for recentering
+   to converge; or it may never converge at all.
```

Comment:
```suggestion
   Depending on the object, it may take several iterations for re-centering
   to converge, or it may never converge at all.
```
One typo fix, and ditches the semi-colon.

Reply:
Good catch. Thanks!